qual_id/categories/drink.py

```diff
@@ -0,0 +1,18 @@
+from ..category import Category
+
+class Drink(Category):
+    def get_values:
```

Comment:
```suggestion
    def get_values(self):
```
Methods need brackets `()`.
Also, this is a class method, and so it needs to take the parameter `self`.

Reply:
Im sorry, i forgot to add it

Reply:
Don't worry about it! If you click 'commit suggestion' it should fix the tests